### DISCUSSION TITLE
Wie am besten Module verbinden und welche Batterieladeregler/Pylontech-Batterie

### DISCUSSION THREAD

**Discussion by maxclaudi** [hoylabs/OpenDTU-OnBattery#1307] [Q&A] [answered]
**vorhanden/IST:**
6x PV-Module:
Trinasolar TSM-440NEG9R.28
Pmax 440W
maximum Power Voltage Vmp 44.0V
maximum Power Current Imp 10.01A
Open Circuit Voltage Voc 52.2V
Short Circuit Current ISC 10,67A

4 Wechselrichter Hoymiles HM-400
2 Wechselrichter Hoymiles HM-350

2x Module nach Süd. Davon 1 Modul teils verschattet von Kamin.
Jeweils HM-400 für jedes Modul

4x Module nach Ost. Davon 1-3 Module teilweise verschattet von Nachbarhaus.
2x Module an jeweils HM-400 und 2 Module an jeweils HM-350

1x AllianceApps OpenDTU Fusion V2.2 
kompatibel mit Hoymiles Wechselrichtern für HM/HMS/HMT USB-C, ESP32-S3, NRF24L01+, CMT2300A, interne Antennen
mit OpenDTU-OnBattery 2024.09.11 generic_esp32s3_usb 

2x OpenDTU Fusion CAN/Iso Shield
um künftige Pylontech Batterie und Victron-Batterieladeregler anzuschließen

1x Shelly Pro 3EM im Zählerschrank verbunden mit OpenDTU-OnBattery

**soll/Einkaufsliste:**
1x Pylontech Batterie
Mehrere Victron Smart-Batterieladeregler

**Probleme:**
Module sind fest installiert und eine bessere Anordnung für weniger Verschattung ist nicht möglich.
Deshalb wäre es gut wenn jedes Modul einen eigenen MPPT-Regler hätte.
Oder zur Not mindestens jeweils 2 Module 1 MPPT.
Die 2 Süd-Module 1 MPPT
Je 2 Ost Module 1 MPPT.
Weil 2 OST Module stärker verschattet sind.

Die Victron-Laderegler haben immer nur 1 MPPT.

**Ziel:**
Überschuss soll Batterie laden.
Batterie voll: volle Einspeisung.
Bei Batterie an unterer Grenze: nur Batterie soll geladen werden.
Nachteinspeisung mit Batterie 150W bis Batterie an unterer Grenze oder Sonnenaufgang. 
Was sind die besten Möglichkeiten?

**2 Ideen:**
1) Jedes Modul mit einem eigenen Victron Smart MPPT 100/20 betreiben mit Pylontech UP2500 2,8kWh 24V Lithium-Batterie
OpenDTU-OnBattery mit 1 Victron verbinden und die Victron untereinander über VE.Smart Netzwerk.
Allerdings wäre mir eine Pylontech 48V lieber, die sollen auch besser für die Hoymiles sein.

2) 2 Module Süd in Reihe
2 Module OST teilverschattet in Reihe
2 Module OST in Reihe 
Die Strings jeweils mit Victron MPPT 150/35  
verbunden mit Pylontech US2000C 2400Wh 48V Lithium-Batterie
Die Victron alle 3, je einzeln mit OpenDTU-OnBattery über UART verbinden.
Auch wenn MPPT wegen Teilverschattung wahrscheinlich nicht so gut regeln kann, wäre mir die 48V Lösung lieber.

Ist das möglich? Was wäre die bessere Option? 

Wer ist so lieb von den Erfahrenen und hilft mir?

**Comment by spcqike** [comment, marked as answer]
Bei so vielen Modulen und unterschiedlichen Ausrichtungen kannst du dir auch Option 3 überlegen. 

In AC gekoppelter Speicher. 

Du lässt deine Hoymiles so, wie sie sind, aber offen in volleinspeisung. Und hast bspw einen Victron multiplus mit 48V Batterie der dann die Überschüsse aufnimmt und Defizite aus der Batterie unterstützt. 

Da gibt es sicher mehrere Möglichkeiten. Nicht nur den Multiplus. 

Ich denke, das wäre meine favorisierte Option bei so vielen und bereits vorhandenen/ installierten Modulen mit unterschiedlicher Ausrichtung / verschattung.

**Reply by maxclaudi** [reply]
Dankeschön

**Reply by spcqike** [reply]
wenn es ein bisschen mehr "plug and play" sein soll (aber eben auch nicht so flexibel und erweiterbar), kannst du dir auch den hoymiles MS-A2 angucken. Dazu gibt es diverse Berichte auf Youtube. 

ein AC speicher mit 2kWh Kapazität. Man kann vorhandene Shellys (scheinbar leider nicht den 3EM, nur den Pro 3EM und Plug S, aber kann ja per Update mehr werden) einbinden und damit ebenfalls die Nulleinspeisung machen.

Ich denke, ich werd später auf Victron Multiplus setzen. Erst einer, für 1-phasig, mit 5 - 10 kWh Batterie, und dann später auf 3-phasig erweitern.

**Comment by Manos1966** [comment]
> Mehrere Victron Smart-Batterieladeregler
Probleme:
Module sind fest installiert und eine bessere Anordnung für weniger Verschattung ist nicht möglich.
Deshalb wäre es gut wenn jedes Modul einen eigenen MPPT-Regler hätte.
Oder zur Not mindestens jeweils 2 Module 1 MPPT.
Die 2 Süd-Module 1 MPPT
Je 2 Ost Module 1 MPPT.
Weil 2 OST Module stärker verschattet sind.
Die Victron-Laderegler haben immer nur 1 MPPT.

"Deshalb wäre es gut wenn jedes Modul einen eigenen MPPT-Regler hätte." 
Wer hat dir so etwas gesagt? Der VICTRON-Verkaeufer?
Empfehlung: Sich mit der Funktionalitaet der Victron MPPT Ladereglern vertraut machen.

Wobei, spcqikes Loesung ist noch einfacher (du muss keine Kabel verlegen)
